services/identity-service/src/main.ts

```diff
@@ -0,0 +1,22 @@
+import { NestFactory } from '@nestjs/core';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
+import { AppModule } from './app.module';
+import { Logger } from '@nestjs/common';
+
+async function bootstrap() {
+  const app = await NestFactory.create(AppModule);
+  app.enableCors()
+  // setup swagger
+  const config = new DocumentBuilder()
+    .setTitle('Identity Service')
+    .setDescription('The Identity Service API description')
+    .setVersion('1.0')
+    .addTag('Identity Service')
+    .build();
+  const document = SwaggerModule.createDocument(app, config);
+  SwaggerModule.setup('api', app, document);
+  const port = process.env.PORT || 3332;
+  await app.listen(port, '0.0.0.0');
+  Logger.log(`🚀 Application is running on: http://localhost:${port}/`);
```

Comment:
change log to 0.0.0.0 instead of localhost

Reply:
resolved